ci/provisioning/post_provision_config_nodes_UBUNTU_20_04.sh

```diff
@@ -52,9 +53,8 @@ post_provision_config_nodes() {
                        python3-avocado-plugins-varianter-yaml-to-mux \
                        lsb-core
 
-    # shellcheck disable=2086
-    if [ -n "$INST_RPMS" ] &&
-       ! apt-get -y install $INST_RPMS; then
+    if [ ${#inst_rpms[@]} -gt 0 ] &&
+       ! apt-get -y install "${inst_rpms[@]}"; then
```

Comment:
(lint) This is actually an end quote, but due to next char it looks suspect. [SC1079]